src/lib/libdylink.js

```diff
@@ -715,7 +715,13 @@ var LibraryDylink = {
           }
           if (prop in wasmImports && !wasmImports[prop].stub) {
             // No stub needed, symbol already exists in symbol table
-            return wasmImports[prop];
+            var res = wasmImports[prop];
+#if JSPI
+            if ('orig' in res) {
```

Comment:
I think maybe just `if (res.orig)` would would here?   (Oh I sees this is copied from line 1250.. maybe update both  of them?)

Reply:
Is this not needed for `ASYNCIFY=1` too?  The version on line 1250 just uses `if ASYNCIFY`.   Can you copy the comment from there too?

Reply:
The test fails with `ASYNCIFY=1` in the same way regardless of this change.

Reply:
Though it looks like the problem is probably fixable by adding another `blah = blah.orig` somewhere else.

Reply:
Okay, I have no idea how to fix this test with `ASYNCIFY`, so I'd prefer to keep it focused on JSPI.